charts/hono/values.yaml

```diff
@@ -1188,6 +1194,7 @@ deviceRegistryExample:
     pod:
       labels: {}
       annotations: {}
+      affinity: {}
```

Comment:
This value still needs to be used in 
`hono-service-device-registry-embedded/hono-service-device-registry-statefulset.yaml`